### ISSUE TITLE
Adding CAN driver example

### ISSUE THREAD

**Issue by RahulGoyal-tech** [gicking/STM8_headers#14]
Some Variants of STM8S comes with a beCAN Module (Basic Enhanced CAN)

Will be adding an example (with mostly a sort of lib) to use beCAN.

Hardware Availability for me : STM8S003F3

**Comment by gicking** [comment]
sorry, I don't quite get your point. Do you want to provide or do you ask for a CAN example?

**Comment by RahulGoyal-tech** [comment]
Will be adding the PR 😅.

**Comment by gicking** [comment]
perfect  :-)

**Comment by gicking** [comment]
> Some Variants of STM8S comes with a beCAN Module (Basic Enhanced CAN)
...
> Hardware Availability for me : STM8S003F3

I just checked and according to the datasheet the STM8S003F3 does not support CAN module

**Comment by RahulGoyal-tech** [comment]
Yeah Found that out. Will be using stm8s207 nucleo board. That have the CAN module
